core/examples/http_bench.rs

```diff
@@ -158,11 +148,12 @@ fn main() {
   });
 
   let isolate = deno_core::Isolate::new(startup_data, false);
-  isolate.register_op("listen", http_op(op_listen));
-  isolate.register_op("accept", http_op(op_accept));
-  isolate.register_op("read", http_op(op_read));
-  isolate.register_op("write", http_op(op_write));
-  isolate.register_op("close", http_op(op_close));
+  let state = HttpBenchState::new();
```

Comment:
it would be good to "game out" how it works out when you do

```rust
struct HttpIsolate {
  core_isolate: deno_core::Isolate,
  state: HttpBenchState,
}
```

Reply:
Sure. We're not accessing the core isolate anywhere so it won't change much I think.